src/server/controllers/collectives.js

```diff
@@ -28,3 +29,107 @@ export async function info(req, res, next) {
 
   res.send(response);
 }
+
+const query = `
+query($slug: String){
+  account(slug: $slug) {
+    memberOf(role: HOST){
+      nodes{
+        account {
+          ... on Collective {
+            name
+            slug
+            description
+            currency
+            isArchived
+            updatedAt
+            approvedAt
+            totalFinancialContributors
+            stats {
+              balance {
+                value
+              }
+              yearlyBudget {
+                value
+              }
+            }
+            members(role: ADMIN) {
+              nodes {
+                account{
+                  ... on Individual {
+                    name
+                    email
+                  }
+                }
+              }
+            }
+          } 
+        }
+      }
+    }
+  }
+}
+`;
+
+export async function hosted(req, res) {
+  try {
+    const response = await getClient({ version: 'v2' }).request(query, { slug: req.params.slug });
```

Comment:
You need to get user credentials (passed from frontend) and forward it to the client, otherwise you will not get restricted data such as emails.

Reply:
Check what we're doing in members.js with `headers.authorization`, that should work. But we need to make sure that Frontend is passing it properly.